src/main/java/nitezh/ministock/activities/MinistocksActivity.java

```diff
@@ -38,6 +38,7 @@ of this software and associated documentation files (the "Software"), to deal
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        GlobalWidgetData.setInterval(2);
         setContentView(R.layout.ministocks);
         Spanned html = Html.fromHtml("Note: This is a widget only app.<br /><br />" + "<b>Getting started</b><br /><br />" + "First long-press an empty space on your Home screen to bring up the <i>Add to Home screen</i> menu.<br /><br />" + "Then select the Widgets option and finally, select the <i>MinistocksActivity</i> item from the list of widgets.<br /><br />" + "Multiple widgets can be added, as each widget stores its own data.<br /><br />" + "<b>Press Home or Back to close.</b><br />");
```

Comment:
`Spanned html = Html.fromHtml("Note: This is a widget only app.<br /><br />" + "<b>Getting started</b><br /><br />" + "First long-press an empty space on your Home screen to bring up the <i>Add to Home screen</i> menu.<br /><br />" + "Then select the Widgets option and finally, select the <i>MinistocksActivity</i> item from the list of widgets.<br /><br />" + "Multiple widgets can be added, as each widget stores its own data.<br /><br />" + "<b>Press Home or Back to close.</b><br />");`

is not styled properly. very difficult to read and should be properly formatted to fit the screen